eru/src/components/modules/cards/createScreelCard.vue

```diff
@@ -133,9 +133,9 @@ export default {
     this.typingPlaceholder();
     //Toggle class every 2 seconds
     setInterval(function () {
-      document
-        .getElementById("create-screel-btn")
-        .classList.toggle("shake-button");
+      const createScreelButton = document.getElementById("create-screel-btn");
+
+      createScreelButton && createScreelButton.classList.toggle("shake-button");
```

Comment:
Yes, the code which was there before was throwing an error in the console because the "document.getElementById('create-screel-btn')" was undefined at times. so I first of all check if the element exists before toggling the "shake-button" class